app/src/main/java/com/twilio/voice/quickstart/VoiceService.java

```diff
@@ -644,6 +645,8 @@ public void onConnected(@NonNull Call call) {
                 soundPoolManager.playSound(SoundPoolManager.Sound.RINGER);
             }
 
+            activeCall = call;
```

Comment:
This violates the underlining design principal of the whole architecture. Notice how all other methods take a UUID? The voice service should not be preserving state like that.